internal/charmstore/migrations_integration_test.go

```diff
@@ -386,6 +450,26 @@ var migrationFromDumpBaseEntityTests = []struct {
 	},
 }}
 
+var migrationFromDumpStatsTests = []struct {
+	id    string
+	total int64
+}{{
```

Comment:
Perhaps it would be nicer if these all had different totals, so we could tell that the right totals were being assigned to their respective entities?